docs/Guides/CommonSolutions.md

```diff
@@ -144,3 +144,49 @@ cp /usr/lpp/cbclib/xlclang/etc/xlclang.cfg /u/fred/xlclang.cfg.zos204.noceertfm
 export CLC_CONFIG=/u/fred/xlclang.cfg.zos204.noceertfm
 ```
 
+## S_TYPEISSHM macro gives an error when compiled
+
+The error might be something like: `called object type 'int' is not a function or function pointer`
+
+This is due to a bug in the LE header file `sys/modes.h` which defines the macros as follows:
```

Comment:
Can we fix this in zoslib by overriding sys/modes.h?

Reply:
If we could fix this in zoslib that would be great.